examples/kafka-streams-examples/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+---
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:3.2.0
```

Comment:
Question: This is a PR against the `3.2.x` branch.  However, when I checked the existing code in that branch, the other examples plus the other tests are always pointing to `:latest` instead of `3.2.0`.  Is this intended?

I am happy to update the labels here, but I feel like only `master` should use `:latest`, no?

(This question applies to a few more such instances where `:3.2.0` is used.)